public/assets/html/createCharacter.html

```diff
@@ -0,0 +1,101 @@
+
+    <!-- Materialize Compiled and minified CSS -->
+    <link rel="icon" href="https://www.nationalgeographic.com/content/dam/animals/thumbs/rights-exempt/mammals/p/plains-zebra_thumb.JPG"
+        sizes="32x32" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/css/materialize.min.css">
+    <title>Create Character</title>
+</head>
+
+<body>
+    <div>
+        <div class="form-group input-field">
+            <label for="namet">Name: </label>
+            <input type="text" id="name" name="name">
+        </div>
+        <br>
+        <h3 id="skillPoints"></h3>
+        <ul class="skills">
+            <li id="Defense" value="20">
+                <h4 class="skillValStr">Defense : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Defense">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Defense">+</button>
+            </li>
+            <li id="Attack" value="20">
+                <h4 class="skillValStr">Attack : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Attack">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Attack">+</button>
+            </li>
+            <li id="Health" value="20">
+                <h4 class="skillValStr">Health : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Health">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Health">+</button>
+            </li>
+        </ul>
+    </div>
+    <br>
+    <button class='btn btn-large waves-effect waves-light grey darken-3 white-text' id="submitBtn">submit</button>
+
+
+    <script>
+        let skillPonints = 20;
+        $("#skillPoints").text("Skill Points: " + skillPonints);
+
+        // increase decrease skill value
+        $(".skills button").on("click", function (event) {
+            event.preventDefault();
+            let skill = $(this).attr("data-skill");
+            let skillVal = parseInt($("#" + skill).attr("value"));
+            if ($(this).text() == "+") {
+                if (skillPonints > 0) {
+                    skillPonints--;
+                    skillVal++;
+                }
+            } else if ($(this).text() == "-") {
+                if (skillVal > 20) {
+                    skillPonints++;
+                    skillVal--;
+                }
+            }
+            $("#skillPoints").text("Skill Points: " + skillPonints);
+            $("#" + skill).attr("value", skillVal.toString());
+            $("#" + skill + " .skillValStr").text(skill + " : " + skillVal.toString());
+        });
+
+        $("#submitBtn").on("click", function (event) {
+            event.preventDefault();
+            let newChar = {
```

Comment:
use object literal syntax when outside of json, no quotes
{
name: ...
attack: ...
...
}